spec/models/user_spec.rb

```diff
@@ -40,34 +40,38 @@
     it "default has the base price group" do
       expect(user.price_groups).to eq [PriceGroup.base]
     end
+
     it "external user has external price group" do
       external_user = FactoryGirl.create(:user, :external)
       expect(external_user.price_groups).to eq [PriceGroup.external]
     end
   end
 
-  it "is a member of any explicitly mapped price groups" do
-    pg = FactoryGirl.create(:price_group, facility: facility)
-    UserPriceGroupMember.create(user: user, price_group: pg)
-    expect(user.price_groups.include?(pg)).to eq(true)
-  end
+  describe "price groups", feature_setting: { user_based_price_groups: true } do
 
-  it "belongs to price groups of accounts" do
-    cc = create(:nufs_account, account_users_attributes: account_users_attributes_hash(user: user))
-    pg = FactoryGirl.create(:price_group, facility: facility)
-    AccountPriceGroupMember.create(account: cc, price_group: pg)
-    expect(user.account_price_groups.include?(pg)).to be true
-  end
+    it "is a member of any explicitly mapped price groups" do
+      pg = FactoryGirl.create(:price_group, facility: facility)
+      UserPriceGroupMember.create(user: user, price_group: pg)
+      expect(user.price_groups.include?(pg)).to eq(true)
```

Comment:
I don't have a strong feeling which way we go, but everywhere else we're using `be true`. We should probably be consistent. Or maybe better change all these to `expect(user.price_groups).to include(pg)`. But again, I'm more interested in getting this wrapped up than spending a ton of time on cleanup.